.NET/Microsoft.Recognizers.Definitions.Common/Spanish/DateTimeDefinitions.cs

```diff
@@ -720,7 +725,7 @@ public static class DateTimeDefinitions
       public static readonly IList<string> MonthToDateTerms = new List<string>
         {
             @"mes a la fecha",
-            @"meses a la fecha"
```

Comment:
Why remove the plural ones?

Reply:
the regex only contain the singular case, and also in English only the case "year to date" is used.